networking_oneview/tests/unit/ml2/drivers/oneview/test_common.py

```diff
@@ -63,3 +63,33 @@ def test_get_oneview_client_insecure_cafile(self, mock_oneview_client):
         self.credentials["ssl_certificate"] = None
         common.get_oneview_client()
         mock_oneview_client.assert_called_once_with(self.credentials)
+
+    def test_check_server_hardware_availability(
+        self, mock_oneview_client
+    ):
+        self.conf.DEFAULT.retries_to_lock_sh_interval = 1
+
+        for i in range(1, 6):
```

Comment:
You can simplify this, doesn't need to create a for loop. And create a positive test.